versioned_docs/version-5.x/nesting-navigators.md

```diff
@@ -105,23 +109,31 @@ In your app, you will probably use these patterns depending on the behavior you
 
 Consider the following example:
 
+<samp id="nested-navigator-screen" />
+
 ```js
 function Root() {
   return (
-    <Stack.Navigator>
-      <Stack.Screen name="Profile" component={Profile} />
-      <Stack.Screen name="Settings" component={Settings} />
-    </Stack.Navigator>
+    <Tab.Navigator>
+      <Tab.Screen name="Home" component={Home} />
+      <Tab.Screen name="Profile" />
+      <Tab.Screen name="Settings" />
+    </Tab.Navigator>
   );
 }
 
 function App() {
   return (
     <NavigationContainer>
-      <Drawer.Navigator>
-        <Drawer.Screen name="Home" component={Home} />
-        <Drawer.Screen name="Root" component={Root} />
-      </Drawer.Navigator>
+      <Stack.Navigator>
+        <Stack.Screen name="Start" component={Start} />
+        <Stack.Screen
+          name="Root"
+          component={Root}
+          options={{ headerShown: false }}
+        />
+        <Stack.Screen name="Feed" />
+      </Stack.Navigator>
```

Comment:
The overall section explains and mentions many times about Drawer navigator so the example here isn't relevant. Either bring back the previous code and create a relevant snack to this (easier) or rewrite the whole section to be relevant to this example (hard)

Personally I would take the first approach 😄